src/main/scala/org/apache/spark/deploy/armada/Config.scala

```diff
@@ -157,6 +157,161 @@ private[spark] object Config {
   private def k8sLabelListValidator(labelList: String): Boolean = {
     parseCommaSeparatedK8sLabels(labelList).forall(_.isSuccess)
   }
+
+  val ARMADA_QUEUE: ConfigEntry[String] =
+    ConfigBuilder("spark.armada.queue")
+      .doc("The Armada queue that will be used for running the driver and executor pods.")
+      .version("1.0.0")
+      .stringConf
+      .createWithDefault("default")
+
+  val DRIVER_JOB_SET_ID: ConfigEntry[String] =
+    ConfigBuilder("spark.armada.driver.job.set.id")
+      .doc("The Armada job set id that will be used for running the driver pods.")
+      .version("1.0.0")
+      .stringConf
+      .createWithDefault("Driver-Job-Set")
```

Comment:
This should be a Spark-app-specific string so that each Spark job has an individual job set in the cluster. This is nothing the user wants to set, is it?

Reply:
During the demo last week, Dexter indicated that he wanted to be able to set it.  Do you want me to remove the config?  Or maybe append a UUID  and use this as a prefix?

Reply:
I think Dexter pointed out the jobset id should be unique for each job where in the demo it was "driver", which implies all Spark jobs will use the same jobset id. Further, it should be the same for driver and executor where in the demo it was "driver" and "executor".

In Spark, the user can define an app name via `--name` or `--conf spark.app.name`.

Spark Kubernetes uses [KubernetesConf.getResourceNamePrefix](https://github.com/apache/spark/blob/aa89883746b06ab7d12fe450e9596520c75f9718/resource-managers/kubernetes/core/src/main/scala/org/apache/spark/deploy/k8s/KubernetesConf.scala#L288-L297) to derive a unique prefix from that app name.

It produces something like `$appName-f4f72a96b5952aff`.